package.json

```diff
@@ -106,6 +106,7 @@
         "testcafe": "1.9.1",
         "ts-jest": "^24.0.2",
         "typescript": "~3.6.0",
+        "wait-port": "^0.2.9",
```

Comment:
Why has this been added as part of this PR?

Reply:
It's a dependency needed to run the `make test_browser` target that's missing, hence why I added it. If you start from a clean checkout of reviewer client, and ensure that you don't have `wait-port` installed globally. If you try to run that target it will fail with...

```
make test_browser
yarn wait-port localhost:9000
yarn run v1.22.4
error Command "wait-port" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
make: *** [test_browser] Error 1
```